sync/templates/controller-vpa.yaml

```diff
@@ -0,0 +1,43 @@
+{{- if .Capabilities.APIVersions.Has "autoscaling.k8s.io/v1" }}
+{{- if .Values.verticalPodAutoscaler.enabled }}
+apiVersion: autoscaling.k8s.io/v1
+kind: VerticalPodAutoscaler
+metadata:
+  name: {{ include "cert-manager.fullname" . }}-vpa
+  namespace: {{ include "cert-manager.namespace" . }}
+  labels:
+    app: {{ include "cert-manager.name" . }}
+    app.kubernetes.io/name: {{ include "cert-manager.name" . }}
+    app.kubernetes.io/instance: {{ .Release.Name }}
+    app.kubernetes.io/component: "controller"
+    {{- include "labels" . | nindent 4 }}
+spec:
+  resourcePolicy:
+    containerPolicies:
+    - containerName: {{ .Chart.Name }}
```

Comment:
I think this doesn't match the container name of the controller
https://github.com/giantswarm/cert-manager-app/blob/main/helm/cert-manager/templates/deployment.yaml#L79

Besides aligning it directly, it might be useful to make a helper for it.